furyctl-defaults.yaml

```
@@ -0,0 +1,192 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+data:
+  # the common section will be used by all the templates in all modules, everything defined here is something used by all the KFD modules.
+  common:
+    # where all the KFD modules are downloaded
+    relativeVendorPath: "../../vendor"
+    provider:
+      # can be eks for now, in the future we will add additional providers
+      type: eks
+    # nodeSelector and tolerations will be used to select the nodes where the KFD will be installed
+    nodeSelector:
+      node.kubernetes.io/role: infra
+    tolerations:
+      - effect: NoSchedule
+        key: node.kubernetes.io/role
+        value: infra
+    # this section will contain the authentication configuration that will be used to protect the KFD infrastructural ingresses
+
+  # the module section will be used to fine tune each module behaviour and configuration
+  modules:
+    # ingress module configuration
+    ingress:
+      overrides:
+        nodeSelector: {}
+        tolerations: []
+        # override ingresses parameters
+        ingresses:
+          forecastle:
+            # disable authentication if set globally on auth module
+            disableAuth: false
+            # if empty, will use the default packageName + baseDomain from common configurations
+            host: ""
+            ingressClass: ""
+
+      baseDomain: example.dev
+      # common configuration for nginx ingress controller
+      nginx:
+        # can be single or dual
+        type: single
+        tls:
+          # can be certManager, secret or none
+          provider: certManager # it uses the configuration below as default when certManager is chosen
+          secret: #if we want to use custom certificates, the template should create a secret and set it as default certificate in NGINX, so patch nginx deployment accordingly
+            cert: |
+              value
+            key: |
+              value
+            ca: |
+              value
+      # the standard configuration for cert-manager on the ingress module
```

Both fields are immutable.
```suggestion
      externalDns:
        privateIamRoleArn: ""
        publicIamRoleArn: ""
      # the standard configuration for cert-manager on the ingress module
```

---

internal